src/javascript/app/pages/trade/barriers.js

```diff
@@ -160,14 +161,73 @@ const Barriers = (() => {
         Defaults.remove('barrier', 'barrier_high', 'barrier_low');
     };
 
-    const validateBarrier = () => {
+    const showError = (barrier) => {
+        if (!barrier.classList.contains('error-field')) {
+            barrier.classList.add('error-field');
+        }
+        const error_node = barrier.parentNode.getElementsByClassName('hint')[0].getElementsByClassName('error-msg')[0];
```

Comment:
Could be a better way to find error_node... Anyone is welcomed to comment for suggestion!

Reply:
do we need this part? `.getElementsByClassName('hint')[0]`

Reply:
Nice! thanks